api/types/types.go

```diff
@@ -528,3 +528,8 @@ type PushResult struct {
 	Digest string
 	Size   int
 }
+
+// BuildResult contains the image id of a successful build
+type BuildResult struct {
```

Comment:
Ah yes, I guess I missed that it was being used by the server as well. My mistake.